CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Climate Connect
+
+Thank you for your interest in contributing to ClimateConnect!
+
+Table of Contents:
+
+1. [Feature Requests](#feature-requests)
+1. [Bug Reports](#bug-reports)
+1. [Pull Requests](#pull-requests)
+    1. [Documentation](#documentation)
+1. [Contact](#contact)
+
+## Feature Requests
+Feature requests should be reported in the [Climate Connect issue tracker](https://github.com/climateconnect/climateconnect/issues). To reduce the number of duplicates, please make sure to check existing issues.
+
+## Bug Reports
+Bug reports should be reported in the [Climate connect issue tracker](https://github.com/climateconnect/climateconnect/issues).
+
+To report a bug follow these guideline.
+
+1. Describe the issue: Fill in a concise description of what went wrong
+1. Steps to recreate: List every step you took that led to the unexpected result.
+1. Expected result: Fill in what you would have expected after the last step in "Steps to recreate"
+1. Actual result: Explain what you actually expected to happen.
+1. Link: Please provide a like to this issue.
+
+## Pull Requests
+If you are looking for a place to start contributing to Climate Connect, take a look at the [beginner level](https://github.com/climateconnect/climateconnect/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+level+issue%22) issues.
```

Comment:
What do you think about adding a couple lines here about what we'd expect from opening a PR? e.g. are tests required?

Reply:
Yeah good point. I will add that.